compute/admin_guide/agentless-scanning/agentless-scanning-modes.adoc

```diff
@@ -28,25 +36,68 @@ endif::prisma_cloud[]
 . Process repeats for periodic scans.
 
+[#hub-account-mode]
 === Hub Account Mode
 
-Scan the workloads and container images of a cloud account (the target) using another cloud account (the hub).
-This mode spins up temporary scanning instances in the hub rather than in the target(s).
+In hub account mode, most of the agentless scanning process takes place in a centralized account known as the hub account.
+You should dedicate this account entirely to the agentless scanning process.
+Each account that the hub account should scan is called a target account.
+
+There is no limit to the number of hub accounts that you can configure and you can use each hub account to scan a different subset of target accounts.
+In this mode, the scanner instances and networking infrastructure are created only on the hub account.
+You don't have to replicate the agentless scanning configuration across target accounts since you can configure agentless scanning centrally on the hub account configuration.
+For example, you don't need to replicate networking configuration across target accounts if you configured your xref:agentless-scanning.adoc#networking-infrastructure[networking infrastructure] in the hub account.
+
+* In AWS, snapshots are created within every target account and are then shared with the hub account.
+* In GCP, snapshots are created directly within the hub account.
+
+Scanners in the hub account scan target accounts independently. An agentless scanner in the hub account only scans snapshots from one target account and this ensures segregation between target accounts.
+
+The following diagram gives a high level view of agentless scanning in hub account mode.
+
+image::agentless-scanning-hub-account-mode.png[width=800]
+
+Agentless scanning in hub account mode consists of the following steps.
 
 ifdef::compute_edition[]
-. Onboard accounts with different permissions for hub account, which perform the scan, and target accounts, which the hub account scans. Follow the step-by-step instructions to xref:./onboard-accounts/onboard-accounts.adoc[configure the permissions].
+. You onboard accounts with different permissions for hub account, which perform the scan, and target accounts, which the hub account scans. Follow the step-by-step instructions to xref:./onboard-accounts/onboard-accounts.adoc[configure the permissions].
 endif::compute_edition[]
 
 ifdef::prisma_cloud[]
-. xref:./onboard-accounts/onboard-accounts.adoc[Onboard your cloud account to Prisma Cloud].
+. xref:./onboard-accounts/onboard-accounts.adoc[You onboard your cloud account to Prisma Cloud].
 endif::prisma_cloud[]
 
 . Prisma Cloud only spins up scanners in the dedicated hub account and attaches snapshots of instances from other accounts to the scanners in the hub account.
 . Scanners send results to the Prisma Cloud Console
 . Scanners then get deleted along with the snapshots that Prisma Cloud creates.
 . Process repeats for periodic scans.
```

Comment:
delete this bulk

Reply:
@omri86 Do you mean all the steps? I can rephrase the introductory sentence and the steps, but I think there is value in still giving customers that perspective. It also makes the differences in the process for same account mode and hub account mode explicitly clear, which helps customers decide which one they want to use.

Reply:
see my comment above